recipes/pyferret/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export LIBZ_DIR=$PREFIX
+export READLINE_DIR=$PREFIX
+export HDF5_DIR=$PREFIX
+export NETCDF4_DIR=$PREFIX
+export FC="gfortran"
+export LD_X11="-L/usr/lib64 -lX11"
+
+export OS=$(echo $(uname) | tr '[:upper:]' '[:lower:]')
+export PLAT=$(uname -p)
+export BUILDTYPE=$PLAT-$OS
+export HOSTTYPE=$BUILDTYPE
+export FER_DIR=$PREFIX
+
+make
+# make run_tests
```

Comment:
The tests are returning a bunch of mismatch images, but they look "OK" to me. Maybe that is a false positive?